### DISCUSSION TITLE
Confirming a gatk command in XBS_call

### DISCUSSION THREAD

**Discussion by abhi18av** [TORCH-Consortium/MAGMA#9] [Q&A] [answered]
Hi @TimHHH ,

Is the following command correct? 

```
 $JAVA -Xmx64G -jar $GATK -V PREFIX.potentialSV.vcf.gz -O PREFIX.potentialSV.DRgenes.vcf.gz -XL $RESOURCE_PATH/DRgenes.list
```

In the original codebase this is commented out at `XBS_call Line-140`. 

I'm confirming this since `gatk TOOL` isn't mentioned, only the `-V and -XL` sub-options are used in the code.

`-V` is same as `--variant`
`-XL` is same as `--exclude-intervals`

**Reference:** 
https://gatk.broadinstitute.org/hc/en-us/articles/360036510892-VariantRecalibrator

**Comment by TimHHH** [comment, marked as answer]
Well spotted.
It should be:
``` $JAVA -Xmx64G -jar $GATK SelectVariants -V PREFIX.potentialSV.vcf.gz -O PREFIX.potentialSV.DRgenes.vcf.gz -L $RESOURCE_PATH/DRgenes.list```
Also note the ```-L``` rather than the ```-XL```.

**Reply by abhi18av** [reply]
Ah, okay. Thanks Tim!
